### DISCUSSION TITLE
worked distances also showing per propagation

### DISCUSSION THREAD

**Discussion by DK5EW** [wavelog/wavelog#848] [Ideas]
Is it possible to show at worked distances tab also per propagation ?
For VHF amateurs it is important to sort this out.

Wini/DK5EW

**Comment by AndreasK79** [comment]
Yeah, that should be doable :)

**Comment by AndreasK79** [comment]
It's in the dev branch now.

**Reply by HB9HIL** [reply]
@DK5EW you can test it in your account on "the special instance" ;-)

**Comment by DK5EW** [comment]
Tested today and it´s a great tool working very well .  Also to find errors in the own logbook. TNX all for implementation !
